### DISCUSSION TITLE
How to see in CLI amount of plots in a pool/nft

### DISCUSSION THREAD

**Discussion by medellin9** [Chia-Network/chia-blockchain#7259] [Support]
I only just started plotting in pools, but I want to verify if things are working as they should.

I saw on some screenshots in the GUI you can see the amount of plots in a pool. But when I do "chia plotnft show" it only shows points/. So how can I see my amount of plots that are in the pool?

And maybe also which ones of my plots are OG or nft plots?

**Comment by loppefaaret** [comment]
seeing the amount of plots on the plotNFT from the CLI, i dont know how to do that yet, sorry
but one way you can differentiate NFT plots from OG plots, is through `chia plots check`, NFT plots will have "pool public key" set to none

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
